src/ResourceManager/Compute/Commands.Compute/help/Invoke-AzureRmVMRunCommand.md

```diff
@@ -13,14 +13,14 @@ Run command on the VM.
 
 ### DefaultParameter (Default)
 ```
-Invoke-AzureRmVMRunCommand [-ResourceGroupName] <String> [-VMName] <String> -CommandId <String>
+Invoke-AzureRmVMRunCommand [-AsJob] [-ResourceGroupName] <String> [-VMName] <String> -CommandId <String>
```

Comment:
How does platyps decide where the parameter is in the parameter set? It seems like -AsJob should not be the first shown in the parameter set.